product_docs/docs/pgd/5/ddl/ddl-overview.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: DDL overview
+navTitle: Overview
+---
+
+DDL stands for data definition language, the subset of the SQL
+language that creates, alters, and drops database objects.
+
+## Replicated DDL
+
+For operational convenience and correctness, PGD replicates most DDL
+actions, with these exceptions:
+
+-   Temporary or unlogged relations
+-   Certain DDL statements (mostly long running)
+-   Locking commands (`LOCK`)
+-   Table maintenance commands (`VACUUM`, `ANALYZE`, `CLUSTER`, `REINDEX`)
+-   Actions of autovacuum
+-   Operational commands (`CHECKPOINT`, `ALTER SYSTEM`)
+-   Actions related to databases or tablespaces
+
+Automatic DDL replication makes 
+certain DDL changes easier without your having to manually distribute
+the DDL change to all nodes and ensure that they're consistent.
+
+In the default replication set, DDL is replicated to all nodes by default.
+To replicate DDL, you must add a DDL replication filter to the
+replication set. See [DDL replication filtering](../repsets/#ddl-replication-filtering).
+
+## Differences from PostgreSQL
+
+PGD is significantly different from standalone PostgreSQL when it
+comes to DDL replication. Treating it the same is the most
+common issue with PGD.
+
+The main difference from table replication is that DDL replication doesn't
+replicate the result of the DDL but the statement itself. This works
+very well in most cases, although it introduces the requirement that the DDL
+must execute similarly on all nodes. A more subtle point is that the DDL
+must be immutable with respect to all datatype-specific parameter settings,
+including any datatypes introduced by extensions (not built in).
+For example, the DDL statement must execute correctly in the default
+encoding used on each node.
+
+## Executing DDL on PGD systems
+
+A PGD group isn't the same as a standalone PostgreSQL server. It's
```

Comment:
This sentence reads almost identical to me as the first sentence of the previous paragraph. I know it isn't exactly saying the same thing but is it not a little redundant?

Reply:
Not really. This is docs for a general audience and it is something we need to hammer home.

Reply:
Got it. I can't see a way to resolve this comment.